lib/profile/sections/section.dart

```diff
@@ -0,0 +1,104 @@
+import 'package:google_fonts/google_fonts.dart';
+import 'package:flutter/material.dart';
+ 
+class Section extends StatelessWidget {  
```

Comment:
Out of curiosity, why did you keep the `SectionRow` and `SectionList` in the same file?

Reply:
I initially thought to place them in separate files but both of them are just a few lines of code so I decided to place them in the Section file since they're part of a section anyways 😅